lib/public/components/Filters/common/filters/floatNumberFilter.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+
+import { selectionDropdown } from '../../../common/selection/dropdown/selectionDropdown.js';
+import { h } from '/js/src/index.js';
+
+/**
+ * Render filter of float number with operator selection
+ *
+ * @param {FloatComparisonFilterModel} floatComparisonFilterModel filter model
+ * @param {object} configuration configuration
+ * @param {string} configuration.selectionPrefix prefix of inputs' identifiers
+ *
+ * @return {Component} the component
+ */
+export const floatNumberFilter = (floatComparisonFilterModel, configuration) => {
```

Comment:
Possibly, but `amountFilter` works with non-negative integers, but for this feature float is required. I would say it would be good to replace `amountFilter` and `floatNumberFilter` by just `numberFilter` implemented like `floatNumberFilter`, but with configurable number of decimal places, but it would require to refactor all components which use `amountFilter`

Reply:
Agreed, maybe create a ticket with low priority to not lose track

Reply:
created O2B-1368